cli/test/org/partiql/cli/CliTest.kt

```diff
@@ -35,7 +37,7 @@ class CliTest {
     private val ion = IonSystemBuilder.standard().build()
     private val output = ByteArrayOutputStream()
     private val testFile = File("test.ion")
-    private val partiqlBagAnnotation = "\$partiql_bag::"
```

Comment:
nit: is this variable necessary? Could we just remove instances of `partiqlBagAnnotation` with `BAG_ANNOTATION`?

Reply:
`MISSING_ANNOTATION` is used directly rather than using another variable.

Reply:
Will update